myhpi/core/templates/core/text_array_widget.html

```diff
@@ -0,0 +1,18 @@
+<link type="text/css" href="/static/css/text_array_widget.css" rel="stylesheet">
+<script src="/static/js/admin/text_array_widget.js"></script>
```

Comment:
I would prefer having these included as django media (https://docs.djangoproject.com/en/5.0/topics/forms/media/#assets-as-a-static-definition)

Reply:
Check out [https://github.com/fsr-de/myHPI/pull/505/commits/4015ded51604ef2b0a6bcf61829946ee8a358862](https://github.com/fsr-de/myHPI/pull/505/commits/4015ded51604ef2b0a6bcf61829946ee8a358862) and feel free to resolve this conversation and merge :)